Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "slog-syslog"
-version = "0.11.0"
-authors = ["Dawid Ciężarkiewicz <dpc@dpc.pw>"]
+version = "2.2.0"
```

Comment:
Why `2.2.0`?

Reply:
We probably want `0.12` instead.

Reply:
The `2.2.0` is to be on par with `slog-json` and `slog-term`

Reply:
I don't think this crate got as much attention and work to deserve 1.0 yet. json and term are only accidentaly at a similiar/same version level.

Reply:
Sounds good I'll roll that back and format it up